### PR TITLE
Zihao/theming and project structure

### PR DESCRIPTION
**Pull request by ltroin**
- Chef overview,tab bar color to black and white
- splash screen provided(not decided what picture yet However could easily replacing with.
- add needed modules, fix bugs, change structure as advised

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,35 +1,51 @@
...
-    "react-native-safe-area-context": "^3.4.0",
-    "react-native-screens": "^3.12.0",
+    "react-native-safe-area-context": "3.3.2",
+    "react-native-screens": "~3.10.1",
+    "react-native-svg": "12.1.1",
     "react-native-vector-icons": "^9.1.0",
-    "react-native-web": "0.17.1"
+    "react-native-web": "~0.17.5",
+    "react-native-webview": "11.15.0",
+    "styled-components": "^5.3.3"
```
Comment (anhanh11001) [inline package.json:38]: I think a lot of dependencies here are not needed, why do you add them?
For example, I think these are not used:`firebase`, `react-dom`, `react-native-web`, `react-native-webview` , but I guess there are more that are not used.

Why did you add them?

**Review by anhanh11001 — COMMENTED** [review]
1. Can you rename test.png to something else like restaurant.png?

Btw, nice changes 👍 it's getting there

File: package.json (hunk 1 above)
Comment (ltroin) [reply package.json:38]: these are librabries used when I tested something, firebase is for login and sign up users, react-native-webview is for rendering an info picture on google maps (because image would not show on google maps on Android ),they are not needed for now

Comment (anhanh11001) [reply package.json:38]: > , firebase is for login and sign up users


But we are using Wix backend, not firebase.

> react-native-webview is for rendering an info picture on google maps (because image would not show on google maps on Android ),they are not needed for now

1. Why do we need webview to render an image, is just normal Image view would work with URL: https://reactnative.dev/docs/image
2. If they are not needed for now, then let's not add it. See YAGNI principle:[ link](https://www.google.com/search?q=yagni&sxsrf=APq-WBv4LkPSwW5wn6SaKB3U2oLtJhSLmQ%3A1646134110122&ei=XgMeYpDbBtGSkwXZs7_QBA&ved=0ahUKEwiQgou356T2AhVRyaQKHdnZD0oQ4dUDCA4&uact=5&oq=yagni&gs_lcp=Cgdnd3Mtd2l6EAMyBAgjECcyBQgAEIAEMgoIABCABBCHAhAUMgoIABCABBCHAhAUMgUIABCABDIFCAAQgAQyBQgAEIAEMgUIABCABDIFCAAQgAQyBQgAEIAEOgcIIxCwAxAnOgcIABBHELADOgcIABCwAxBDOgoIABDkAhCwAxgAOg8ILhDUAhDIAxCwAxBDGAE6BwgjEOoCECc6CwguEIAEEMcBENEDOggILhCABBDUAjoHCAAQgAQQCjoECAAQQzoFCC4QgARKBAhBGABKBAhGGAFQiQlYg5cBYPKYAWgEcAF4AIABdogB3gOSAQM0LjGYAQCgAQGwAQrIARPAAQHaAQYIABABGAnaAQYIARABGAg&sclient=gws-wiz)


Btw it is not the 4 dependencies above, but a lot of others that I find unused
